### ISSUE TITLE
Remove Post Icons

### ISSUE THREAD

**Issue by WillStrohl** [DNNCommunity/Dnn.CommunityForums#160]
## Is your feature request related to a problem?
There is a feature that allows end-users to optionally select an icon for their post.  This might be have been a good feature in the past, but this is an extra step that doesn't appear to offer any true value to any stakeholder.  It's more work, barely changes anything, doesn't help to create/resolve conversations, and so on.  

## Describe the solution you'd like  
Unless we can get someone to provide a current use case for this feature, we should remove it and all related code/schema that supports it.  

## Describe alternatives you've considered
None at this time.  

## Additional context
None at this time.

**Comment by Timo-Breumelhof** [comment]
Agreed, I think selecting a pre-defined category (which renders a preset icon) could be useful, but not just a custom icon
